packages/medusa/src/models/product-variant-money-amount.ts

```diff
@@ -0,0 +1,21 @@
+import { BeforeInsert, Column, Entity, Index } from "typeorm"
+import { generateEntityId } from "../utils"
+import { SoftDeletableEntity } from "../interfaces"
+
+@Entity()
+export class ProductVariantMoneyAmount extends SoftDeletableEntity {
```

Comment:
question: is this table the link between product and pricing modules?
I believe the link will be with price_set, but in case this is the link module's pivot table, we'll need the column `id`

Reply:
This table does have an `id` column as it extends softDeletable, is this what you mean?

Reply:
Yes, it is. 🤦🏻‍♂️ sorry haha. I didn't see the `beforeInsert`.